docs/hub/security-gpg.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I feel this page is a bit too long, maybe consider splitting it up

Reply:
I'd thought about that too, but since it's a guide I think splitting it up would be redundant because users would be going through all of the steps. If it still feels too long when we look at it with the new docs structure then I can totally split it up